packages/nodes-base/nodes/HomeAssistant/HomeAssistant.node.json

```diff
@@ -0,0 +1,21 @@
+{
+	"node": "n8n-nodes-base.homeAssistant",
+	"nodeVersion": "1.0",
+	"codexVersion": "1.0",
+	"categories": [
+		"DIY",
+		"IOT"
```

Comment:
These are the available categories:

<img width="420" alt="Screenshot 2021-06-02 at 3 11 33 PM" src="https://user-images.githubusercontent.com/44588767/120487622-6b80cf00-c3b6-11eb-8049-6115c34c0988.png">

Reply:
changed it to `Miscellaneous` category